lib/util/abi_crosswalk.json

```diff
@@ -2231,6 +2231,26 @@
     "node_abi": 72,
     "v8": "7.8"
   },
+  "12.22.8": {
```

Comment:
since https://github.com/mapbox/node-pre-gyp/pull/711 was merged, I guess you could rebase against `master` and drop this file from the PR